static/app/components/modals/debugFileCustomRepository/appStoreConnect/index.tsx

```diff
@@ -59,9 +60,8 @@ type InitialData = {
   itunesSession: string;
   itunesUser: string;
   name: string;
-  orgId: number;
+  orgPublicId: number;
```

Comment:
it looks like `orgId` is only present in app store connect repos, and is used in this one spot here. re-authentication was failing because the server was no longer returning `orgId`, but `orgPublicId` instead.

as much as the client/FE doesn't need to care about whether the ID is public or not, it looks like it'd be non-trivial to deserialize the `symbolSource` project option, find the appconnect sources, rename their `orgPublicId`s to `orgId`s, and then re-serialize them just to allow the FE to use `orgId` instead of `orgPublicId`.